### DISCUSSION TITLE
Why does <Input type="file" /> shows the native browser implementation?

### DISCUSSION THREAD

**Discussion by MrSnix** [zeit-ui/react#466] [Q&A]
As the title says, I was expecting something more "geist" 😄 
I didn't find anything in the doc about `file input` (I hope I didn't miss it 😭 ).
<img src="https://user-images.githubusercontent.com/20780192/109392210-29f60900-791b-11eb-8df0-5cdf33546830.png" width="300">

I was able to use a button to wrap around the native `input type="file"` and came up with a custom implementation:
<img src="https://user-images.githubusercontent.com/20780192/109392047-40e82b80-791a-11eb-8a13-4cd6e654734e.gif" width="500">

I wonder if something like this can be exposed natively by geist.

Thanks 😃

**Comment by unix** [comment]
Hmmm, did miss that 😿 , now the experience doesn't look too good.
It can also be added to the next version of the optimization, I will come back to remind you when there is an update.

**Reply by Deep-Codes** [reply]
How does this Look like?

![file](https://user-images.githubusercontent.com/61158210/113174832-698c7980-9268-11eb-9203-a78d23d6cd9e.gif)

**Reply by unix** [reply]
Looks good, I think it's feasible.

**Reply by Deep-Codes** [reply]
Yeah, I can try a PR on this.

**Reply by MrSnix** [reply]
Any news about this?

**Reply by MrSnix** [reply]
Hi @unix, has this been addressed ?
